docs/PID tuning.md

```diff
@@ -1,33 +1,55 @@
-# PID tuning
+# Controller introduction
 
-Every aspect of flight dynamics is controlled by the selected "PID controller". This is an algorithm which is
-responsible for reacting to your stick inputs and keeping the craft stable in the air by using the gyroscopes and/or
-accelerometers (depending on your flight mode).
+The model of a controller as used by Cleanflight can be descriped as a pice of software that is designed to minimize errors. In general the controller compares the desired vaule which is given by the RC inputs against the actual value which is sensed by the numerous sensors. Thereby it calculates the deviation - further called Error. This error is then fed into a algorithm that has some knowledge of the system it controls (the aircraft). The end-product or rather the output of the controller is the control value that is designated to correct the error. So the parameters given in the configurator screen (also often referenced as "PIDs") can be imagined as the "knowledge" of the system by the controller. That means that the optimal PID settings to use are different on every craft.
 
-The "PIDs" are a set of tuning parameters which control the operation of the PID controller. The optimal PID settings
-to use are different on every craft, so if you can't find someone with your exact setup who will share their settings
-with you, some trial and error is required to find the best performing PID settings.
+##PIDs
+
+The acronym PID stands for the three main components of the controller: Proportional, Integral and Derivative. These components further called Terms are complete controllers by itself but work together to find the best possible output value to correct the sensed error. Each term uses the tuning factor as a gain for its algorithm. Therefore the terms can be disabled to isolate problems by setting the corresponding gain to 0.
+
+**The P term** is a simple controller. It only does a multiplication of the error with its gain. Therefore it is not time variant and gives the fastest possible response to an error. On the other hand the P term is unprecise. There must be an error for the P term to become active. The maximum error in steady state can be reduced by increasing the P gain but at some point the system will get instable. P Oszillations will generally manifest as higher frequency (quicker) oscillation around the affected axis.
+
+**The I term** also multiplies the error with its gain but in addition sums the product over time. This time variant behaviour causes the I term to be more susceptible to overshouting and oscillations. Hence the I term is tuned way lower than the P term in most cases. The gain is that even small errors can lead to great output values after some time making the I term good for slow, precise corrections. I oscillations a generally distinctively slower that P oscillations (low frequency).
```

Comment:
typo 'overshouting' -> 'overshooting'.  

change 'way lower' -> 'much lower' - it will translate better for non-english readers.

`I oscillations a generally distinctively slower that P oscillations (low frequency).` ??  needs correction.